doc/src/build/objects.md

```diff
@@ -60,6 +60,19 @@ The set of objects that are available to be taken as input by a transaction are
 
 When this DAG contains all committed transactions in the system, it forms a complete (and crytographically auditable) view of the system's state and history. In addition, we can use the scheme above to construct a DAG of the relevant history for a subset of transactions or objects (e.g., the objects owned by a single address).
 
+## The roles of Narwhal and Tusk
```

Comment:
I think "authorities", "transactions", or a separate doc on "consensus" would be a better home for this. The other content in objects is at a much lower level of abstraction--objects are the data model, and the data model is agnostic to the consensus we use.

Reply:
Ack. Thanks for the context. Moving this to Authorities instead in:
https://github.com/MystenLabs/sui/pull/1263